tests/Builder/Expression/Pipelines.php

```diff
@@ -518,6 +518,27 @@ enum Pipelines: string
     ]
     JSON;
 
+    /**
+     * Example
```

Comment:
Do these names have any value? It looks like they're already incorporated into the `case` identifier (after a transformation of the base filename). Considering removing them in a subsequent PR. I agree the `@see` links are relevant, though.

Also, I forget if this ever came up in an earlier PR but it'd be good to check that the generator enforces unique test names per file -- unless you're intentionally relying on PHP to alert you to duplicate `case` IDs when failing to parse this generated file.

Reply:
It's all generated, there is no cost of having this names, and it may be more readable than the CamelCasedCaseName.

I rely on the php generator to check there is no duplicate name. Also PHP throw an error if there is a duplicate value in enum cases (benefit of using an enum over class constants).